MycroftAI/mycroft-core
# Feature/log merger

## Description
Add a new script to merge Mycroft process log files into a single file for debugging purposes.  Also reformats the log output.

## How to test
Start Mycroft processes and see that date and other new formatting is present in log files.  Run the script against log files on a device to see the merged output.

## Contributor license agreement signed?
CLA [yes]
